search-app/src/cmr/search/api/variables.clj

```diff
@@ -30,7 +30,7 @@
   "Verifies the current user has been granted permission to make variable associations"
   [context permission-type]
   (when-not (seq (acl/get-permitting-acls
-                  context :system-object "INGEST_MANAGEMENT_ACL" permission-type))
+                  context :provider-object "INGEST_MANAGEMENT_ACL" permission-type))
```

Comment:
Do we have to make this change right away?

I think this will make it so that anyone that has permission to any provider can associate variables from any provider to collections for any provider.

I would like to see an answer to Leo's email about whether we can limit associations to within a single provider. If so I think we should make the change at that point. Validate that the user has ingest update permissions for that single provider.